config.js

```diff
@@ -514,6 +514,18 @@ config.DB_TYPE = process.env.DB_TYPE || 'mongodb';
 ////////////////////////////////
 config.NAMESPACE_MONITOR_DELAY = 3 * 60 * 1000;
 
+
+////////////////////////////////
+//      BUCKET REPLICATOR     //
+////////////////////////////////
+
+config.BUCKET_REPLICATOR_DELAY = 15 * 60 * 1000;
+config.BUCKET_REPLICATOR_LIST_LIMIT = 1000;
+config.BUCKET_REPLICATION_MAX_RULES = 1000;
+config.BUCKET_REPLICATION_MIN_RULES = 1;
```

Comment:
why do we need MIN for replication rules? will we want to enforce more than one rule

Reply:
same, for compatibility with S3 bucket replication
'A replication configuration must include at least one rule, and can contain a maximum of 1,000. '
https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketReplication.html

Reply:
I think that we will never want to change it to something that is not 1, so adding a config for that just makes it a bit confusing and harder to read when looking at the code

Reply:
removed from config